README.md

```diff
@@ -113,7 +114,9 @@ List of variables which are not defined with default values in ansible role. How
 | `teku_validators_graffiti`                        | `validators-graffiti`                        | Graffiti to include during block creation (gets converted to bytes and padded to Bytes32)   |
 | `teku_validators_keystore_locking_enabled`        | `validators-keystore-locking-enabled`        | Enable locking validator keystore files (Valid values True, False)                          |
 | `teku_validators_performance_tracking_enabled`    | `validators-performance-tracking-enabled`    | Enable validator performance tracking and logging (Valid values True, False)                |
+| `teku_validators_early_attestations_enabled`      | `validators-early-attestations-enabled`      | Enable early attestation production (Valid values True, False)                              |

```

Comment:
Should the second column here be true/false?

Reply:
Second column is what the config file item name is.